packages/runtime/src/useCases/transport/relationshipTemplates/LoadPeerRelationshipTemplate.ts

```diff
@@ -1,21 +1,54 @@
 import { Result } from "@js-soft/ts-utils";
-import { AccountController, RelationshipTemplateController, Token, TokenContentRelationshipTemplate, TokenController } from "@nmshd/transport";
+import { AccountController, RelationshipTemplateController, RelationshipTemplateReference, Token, TokenContentRelationshipTemplate, TokenController } from "@nmshd/transport";
 import { Inject } from "@nmshd/typescript-ioc";
+import { nameof } from "ts-simple-nameof";
 import { RelationshipTemplateDTO } from "../../../types";
-import { Base64ForIdPrefix, RelationshipTemplateReferenceString, RuntimeErrors, SchemaRepository, SchemaValidator, TokenReferenceString, UseCase } from "../../common";
+import {
+    Base64ForIdPrefix,
+    RelationshipTemplateReferenceString,
+    RuntimeErrors,
+    SchemaRepository,
+    SchemaValidator,
+    TokenReferenceString,
+    UseCase,
+    ValidationFailure,
+    ValidationResult
+} from "../../common";
 import { RelationshipTemplateMapper } from "./RelationshipTemplateMapper";
 
 /**
  * @errorMessage token / relationship template reference invalid
  */
 export interface LoadPeerRelationshipTemplateRequest {
     reference: TokenReferenceString | RelationshipTemplateReferenceString;
+    password?: string;
+    pin?: string;
```

Comment:
as there has to be made a difference between pin and password when **creating** an object to make it obvious for the consumer to enter a correct pin/password we should simplify the behavior here, by simply asking for a password. Same for other `load*` UseCases.